XenAdmin/Controls/CustomDataGraph/DataSet.cs

```diff
@@ -99,6 +99,7 @@ public DataSet(IXenObject xo, bool hide, string datasourceName, List<Data_source
                 case "hits/s":
                 case "misses/s":
                 case "err/s":
+                case "sessions/s":

```

Comment:
I noticed that Toolstack have set the units to `sessions per second`. I've asked them to change it to be consistent with the cases above.

Reply:
Already changed it myself and the pr has been merged :)